### DISCUSSION TITLE
About OpenAI Devday 1 event updates - November 2023

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/openai#411] [General]
Hi all, I know you all are as excited as I am about new features and updates. I have already seen a couple of PRs, and I appreciate the support. I hope to be available to work on these next week and release a new version of the library as soon as possible. However, I will be a little slow this time in catching up with the new features because I need to move to a new flat soon(again 😢). I apologize for any inconvenience caused.

I will probably release 2 versions. The first version will have the API/models that can be quickly integrated, while the second one will include more breaking changes and support new features that require more planning.

If you have any suggestions, questions, or feedback, please let's discuss them under this topic.

**Comment by belaszalontai** [comment]
DALL-E-3 is done. I have started to work on Text To Speech API based on this ref. https://platform.openai.com/docs/api-reference/audio/createSpeech

**Reply by belaszalontai** [reply]
**tts-1** feature is implemented, @kayhantolga please merge master to dev then I will create a pull request

**Reply by kayhantolga** [reply]
merged

**Reply by belaszalontai** [reply]
PR is placed, please review.

**Comment by nqnii** [comment]
When can it be done?🙏

**Reply by kayhantolga** [reply]
My plan is to implement all features next week.

**Reply by belaszalontai** [reply]
Vision PR is placed please review..
